openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -94,7 +94,7 @@ def test_get_user_programs_credentials(self):
         self.mock_credentials_api(self.user, reset_url=False)
 
         actual = get_user_program_credentials(self.user)
-        expected = self.PROGRAMS_API_RESPONSE['results']
+        expected = self.PROGRAMS_API_RESPONSE['results'][:2]
```

Comment:
@schenedx the test credentials data will only match up with the first two test programs. This test will fail if you remove this.